### DISCUSSION TITLE
Wallabag

### DISCUSSION THREAD

**Discussion by marapavelka** [FreshRSS/FreshRSS#4150] [Q&A] [answered]
How can I add articles to Wallabag? I know I can set it up via Sharing, but I don't know what to enter as an URL. I'm using docker to selfhost Wallabag.

**Comment by Frenzie** [comment]
Just the URL without any extras like <https://github.com/>. It will then automatically add `?action=add&amp;url=https://linktoarticle.com/blahblah` at the end for when you click on it, which will pass on the URL to your Wallabag instance. (It doesn't even have to be publically accessible for that to work, as long as the link works in your local browser.)

**Comment by marapavelka** [comment, marked as answer]
I tried that, of course, but I got a 404 error. Eventually, I managed to solve it for wallabag v2, the URL must not contain a trailing slash, I didn't notice it last night :) Wallabag v1 doesn't work though, it only opens the page https://wallabag.mydomain.com/?action=add&url=https%3A%2F%2F... but it doesn't save the article. But it doesn't matter, as long as it works with wallabag v2 :D

**Reply by Frenzie** [reply]
Oh right, I mentioned what the wallabag v1 option does. The wallabag v2 does `/bookmarklet?url=etc`. But which to pick should normally be fairly self-explanatory. ^_^

**Comment by math-GH** [comment]
Where should we document this information?

**Reply by Alkarex** [reply]
Maybe a new section in our documentation

**Reply by math-GH** [reply]
see https://github.com/FreshRSS/FreshRSS/pull/4326

**Comment by LnLcFlx** [comment]
This discussion is quite old, but in case anyone is interested in an automatic solution for uploading to Wallabag, I'll leave this here:
- #7227
